README.md

```diff
@@ -70,6 +183,61 @@ Here's the `settings.json` file we are using for configuring the vscode:
   "python.defaultInterpreterPath": "/Users/polywrap/Library/Caches/pypoetry/virtualenvs/polywrap-client-abcdef-py3.10"
 }
 ```
-You need to put your virtual env path you got from the poetry under: `python.defaultInterpreterPath`
 
-Once you configure this you should be good to go.
+Keep in mind that these venv paths will vary for each module you run `poetry shell` on. Once you configure these `setting.json` files correctly on each module you should be good to go!
+
+
+# What WASM wrappers can you execute today?
+
+Check these resources to browse a variety available wrappers, for DeFi, decentralised storage, and other development utilites:
+
+- [Wrappers.io](https://wrappers.io/)
+- [Polywrap Integrations Repository](https://github.com/polywrap/integrations)
+
+# Example call
+
+Calling a function of a wrapper from the python client is as simple as creating a file in the `TODO (?polywrap-client)` directory, importing the Polywrap Python Client, calling the Uri where the WASM wrapper is hosted, and specifying any required arguments.
```

Comment:
Let's have a working call with the hello world or simple subinvoke wrapper we have

Reply:
updated to simple hello world for now, let me know if the syntax would be correct in this case